src/models/bookingBy.ts

```diff
@@ -0,0 +1,9 @@
+export class BookingBy {
```

Comment:
Is this object for Reservation request? If so, let's change the name to something more meaningfull